spec/cpp_interface.dd

```diff
@@ -315,7 +315,7 @@ instead disabled on the D side; an alternative would be to reimplement the
 constructor in D. See the $(DDSUBLINK spec/cpp_interface, lifetime-management,
 section below on lifetime management) for more information.)
 
-$(H3 Using D Classes From C++)
+$(H3 $(LNAME2 using_d_classes_from_cpp, Using D Classes From C++))
 
     $(P Given D code like:)
 
```

Comment:
Can you give "Comparing D Immutable and Const with C++ Const" the same treatment?